src/CacheKernel.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace App;
+
+use Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache;
+
+class CacheKernel extends HttpCache
```

Comment:
don't create a class if you don't add anything to it.
You can directly use HTTPCache in index.php.
In the future if you need something specific to change you could create this child class (CacheKernel)